### ISSUE TITLE
Aria Live Announcer

### ISSUE THREAD

**Issue by emilio-martinez** [emilio-martinez/emc-webcomponents#2]
An element that helps with announcing text through screen readers. It should be a singleton service that should support:
- _message_: Message to be announced to the screen reader.
- _politeness_: Level of politeness or degrees of importance with which the message should be announced.

## Tasks
- [ ] Create element
- [ ] Ensure singleton service
- [ ] Unit tests
- [ ] E2E tests
- [ ] Browser/device compat
- [ ] Demo page

## References
- [WAI-ARIA aria-live](https://www.w3.org/TR/wai-aria-1.1/#aria-live)
- [Angular CDK LiveAnnouncer](https://github.com/angular/material2/tree/master/src/cdk/a11y/live-announcer)
